config/config.py

```diff
@@ -64,12 +64,21 @@ def build_config(
                 }
             ),
             "PU_reweighting_hist": "pileup",
-            "met_filters": ["Flag_goodVertices", "Flag_METFilters"],
+            "met_filters": [               
+                "Flag_goodVertices", 
+                "Flag_globalSuperTightHalo2016Filter", 
+                "Flag_HBHENoiseFilter", 
+                "Flag_HBHENoiseIsoFilter",
+                "Flag_EcalDeadCellTriggerPrimitiveFilter", 
+                "Flag_BadPFMuonFilter", 
+                "Flag_ecalBadCalibFilter",  # only for 2017/18
+                "Flag_eeBadScFilter",
+            ],
         },
     )
     # Tau base selection:
     configuration.add_config_parameters(
-        "global",
+        ["global","mt","tt", "et"],
```

Comment:
i would keep this seperate, so the cuts for `global` only and add a section down below, there the base selections for the three channels are set seperately. e.g. in `tt` we have a pt cut of 40 for the taus, so we will need it anyways

Reply:
I had a problem with this, since I need `"tau_sf_file"` in `global` for the tau energy correction and also in the different channels for calculating the scale factors. I didn't want to have it twice in the config.